tests/test_contrib/test_missing/test_bar.py

```diff
@@ -31,29 +31,37 @@
 ## Feature Importances Tests
 ##########################################################################
 
-class TestMissingBarVisualizer(VisualTestCase):
+
+@pytest.mark.usefixtures("missingdata")
+class MissingBarVisualizerTestCase(VisualTestCase):
     """
-    FeatureImportances visualizer
+    MissingBar Visualizer
     """
 
     def setUp(self):
-        super(TestMissingBarVisualizer, self).setUp()
+        super(MissingBarVisualizerTestCase, self).setUp()
         self.tol = 0.01
         if os.name == 'nt': # Windows
             self.tol = 0.5
```

Comment:
In another PR, I'm going to move these tolerance settings into `VisualTestCase` so they can be more generally shared.